src/wallet/rpcwallet.cpp

```diff
@@ -3539,6 +3545,8 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
             "  \"scriptPubKey\" : \"hex\",       (string) The hex-encoded scriptPubKey generated by the address\n"
             "  \"ismine\" : true|false,        (boolean) If the address is yours or not\n"
             "  \"iswatchonly\" : true|false,   (boolean) If the address is watchonly\n"
+            "  \"issolvable\" : true|false,    (boolean) Whether we know how to spend coins sent to this address, ignoring the possible lack of private keys\n"
```

Comment:
I think it would be nicer to call this "solvable" instead of "issolvable" for consistency with `listunspent`. This would be less consistent with other fields here like "iswatchonly", but I think it's worse to have different names for the same field than different styles of names for different fields.

Reply:
Done.